data_processing/maq_transform_data.R

```diff
@@ -60,60 +77,147 @@ process_maq <- function(rawdata) {
+            uses_videochat_see_parents = sum(
+                str_detect(MediaUseAttitudes.ol.reason, "see parents") &
+                    str_detect(
+                        MediaUseAttitudes.ol.levelofagreement,
+                        "Sometimes|Often|Very often|Very Frequently"
+                    )
+            ) > 0,
+            uses_videochat_see_parents_text = paste0(MediaUseAttitudes.ol.levelofagreement[str_detect(MediaUseAttitudes.ol.reason, "see parents")], collapse = ", "),
+            uses_videochat_connect_day = sum(
+                str_detect(MediaUseAttitudes.ol.reason, "throughout the day") &
+                    str_detect(
+                        MediaUseAttitudes.ol.levelofagreement,
+                        "Sometimes|Often|Very often|Very Frequently"
+                    )
+            ) > 0,
+            uses_videochat_connect_day_text = paste0(MediaUseAttitudes.ol.levelofagreement[str_detect(MediaUseAttitudes.ol.reason, "throughout the day")], collapse = ", "),
+            uses_videochat_connect_work = sum(
+                str_detect(MediaUseAttitudes.ol.reason, "because of work schedule") &
+                    str_detect(
+                        MediaUseAttitudes.ol.levelofagreement,
+                        "Sometimes|Often|Very often|Very Frequently"
+                    )
+            ) > 0,
+            uses_videochat_connect_work_text = paste0(MediaUseAttitudes.ol.levelofagreement[str_detect(MediaUseAttitudes.ol.reason, "because of work schedule")], collapse = ", ")
         ) %>%
-        select(child_id, sf_maq_Q10_uses_videochat_b)
+        select(
+            child_id,
+            uses_videochat_relatives,
+            uses_videochat_relatives_text,
+            uses_videochat_see_parents,
+            uses_videochat_see_parents_text,
+            uses_videochat_connect_day,
+            uses_videochat_connect_day_text,
+            uses_videochat_connect_work,
+            uses_videochat_connect_work_text
+        )
     
     ######
-    ## Media device use
+    ## Media device use/ Parent Attitudes
     ######
-
+    
     mediadeviceuse = recombine(list("Respondents", "MediaDeviceUse"), rawdata) %>%
         full_join(children, by = 'respondent_id') %>%
         group_by(child_id) %>%
         summarise(
-            sf_maq_Q11_avoid_media_for_calming = sum(str_detect(MediaDeviceUse.ol.reason, "calm child down") & str_detect(MediaDeviceUse.general.frequency, "Never")) > 0
+            sf_maq_Q11_avoid_media_for_calming = sum(
+                str_detect(MediaDeviceUse.ol.reason, "calm child down") &
+                    str_detect(MediaDeviceUse.general.frequency, "Never")
+            ) > 0,
+            calming_down = paste0(MediaDeviceUse.general.frequency[str_detect(MediaDeviceUse.ol.reason, "calm child down")], collapse = ", "),
+            avoid_media_for_educating = sum(
```

Comment:
maybe we should turn this into `use_media_for_educating` (sum > 0), since the others are more _negative_ actions and this one is positive?

Reply:
yeah that makes sense!